homeassistant/components/webostv/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "LG webOS Smart TV",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/webostv",
-  "requirements": ["aiopylgtv==0.4.0"],
+  "requirements": ["bscpylgtv==0.2.7"],
```

Comment:
There's a problem in the library. It pins websockets to an exact version. We don't want that.

https://github.com/chros73/bscpylgtv/blob/2487f498c808c5d2193e9ec625e1b031943006f3/setup.py#L17

Can we ask them to adjust the library to be compatible with latest websocket version and just require a minimum version?

Reply:
I asked, will updated when I have reply

Reply:
Package was updated:
https://github.com/chros73/bscpylgtv/blob/c8b8251684215a5add80e1bb706bf1d31b6e194f/setup.py#L17
`install_requires=["websockets>=8.1", "sqlitedict"],`

Reply:
Great!